src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.Description": "Ce formulaire vous permet de personnaliser votre calendrier de jeux en précisant l'année en cours, le nombre de mois, le nom de chaque mois et le nombre de jours de chaque mois.",
+  "FSC.Configuration.Help": "Pour obtenir de l'aide sur la configuration du calendrier, assurez-vous de consulter la <a href='https://github.com/vigoren/foundryvtt-simple-calendar/tree/main/docs/Configuration.md ' target='_blank'>documentation!</a>",
+  "FSC.Configuration.Save": "Enregistrer la configuration",
+  "FSC.Configuration.General.Title": "Paramètres généraux",
+  "FSC.Configuration.General.PreDefined": "Calendriers prédéfinis",
+  "FSC.Configuration.General.PreDefinedHelp": "Choisissez l'un des calendriers prédéfinis pour démarrer la configuration plus rapidement.",
+  "FSC.Configuration.General.GameWorldTime": "Intégration du temps dans le monde du jeu",
+  "FSC.Configuration.General.GameWorldTimeHelp": "Dites à Simple Calendar comment s'intégrer au temps du monde de jeu et répondre aux mises à jour du temps du monde de jeu.",
+  "FSC.Configuration.General.None": "Aucun : N'est pas lié au temps du monde de jeu.",
+  "FSC.Configuration.General.Self": "Self: Contrôlez le temps du monde de jeu en ignorant les mises à jour des autres modules.",
+  "FSC.Configuration.General.ThirdParty": "Module tiers : Suit uniquement les mises à jour du temps du monde de jeu provenant d'autres modules.",
+  "FSC.Configuration.General.Mixed": "Hybride : Mettre à jour le temps du monde de jeu et suivre les mises à jour effectuées à partir d'autres modules.",
+  "FSC.Configuration.General.PF2ESync": "Pathfinder 2E : synchronisation du temps du monde de jeu",
+  "FSC.Configuration.General.PF2ESyncHelp": "Cela forcera Simple Calendar à rester synchronisé avec le temps du monde de jeu intégrée de Pathfinder 2E.",
+  "FSC.Configuration.General.ShowClock": "Afficher l'horloge",
+  "FSC.Configuration.General.ShowClockHelp": "Si pour afficher l'horloge sous le calendrier, si coché, les commandes pour changer les heures, les minutes et les secondes s'afficheront également.",
+  "FSC.Configuration.General.PlayerAddNotes": "Ajouter des notes",
+  "FSC.Configuration.General.PlayerAddNotesHelp": "Les rôles cochés pourront ajouter des notes au calendrier.",
+  "FSC.Configuration.General.PlayerReorderNotes": "Réorganiser les notes",
+  "FSC.Configuration.General.PlayerReorderNotesHelp": "Les rôles cochés pourront réorganiser les notes un jour.",
+  "FSC.Configuration.General.QuickSetup": "Configuration rapide",
+  "FSC.Configuration.General.Notes": "Notes",
+  "FSC.Configuration.General.ImportExport.Title": "Importer/Exporter",
+  "FSC.Configuration.General.ImportExport.SCConfig": "Importer/Exporter la configuration du calendrier",
+  "FSC.Configuration.General.ImportExport.SCConfigHelp": "Pour importer ou exporter la configuration actuelle du calendrier Simple Calendar pour partager ou charger la réplication dans un autre monde.",
```

Comment:
We could avoid repetition and adjust the translation for this:
"Pour importer ou exporter la configuration actuelle **de** Simple Calendar pour partager ou charger la réplication dans un autre monde."

The original text seems weird to me (or I am getting too tired) Maybe it should be reworded.

Reply:
I don't understand where you are going with this. The version I suggest simply replaces "configuration actuelle du calendrier Simple Calendar" to "configuration actuelle de SImple Calendar"